core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -274,44 +274,39 @@ var StoryEditorPage = function() {
   };
 
   this.changeStoryTitle = async function(storyTitle) {
-    await storyTitleField.clear();
-    await storyTitleField.sendKeys(storyTitle);
+    await action.clear('Story Title Field', storyTitleField);
+    await action.sendKeys('Story Title Field', storyTitleField, storyTitle);
   };
 
   this.returnToTopic = async function() {
     await general.scrollToTop();
-    await returnToTopicButton.click();
+    await action.click('Return To Topic Button', returnToTopicButton);
     await waitFor.pageToFullyLoad();
   };
 
   this.changeStoryDescription = async function(storyDescription) {
-    await storyDescriptionField.clear();
-    await storyDescriptionField.sendKeys(storyDescription);
+    await action.clear('Story Description Field', storyDescriptionField);
+    await action.sendKeys(
+      'Story Description Field', storyDescriptionField, storyDescription);
   };
 
   this.changeStoryNotes = async function(richTextInstructions) {
-    await openStoryNotesEditorButton.click();
+    await action.click(
+      'Open Story Notes Editor Button', openStoryNotesEditorButton);
     var storyNotesEditor = await forms.RichTextEditor(
       notesEditor);
-    await storyNotesEditor.clear();
+    await action.clear('Story Notes Editor', storyNotesEditor);
```

Comment:
`storyNotesEditor` is a RichTextEditor, not an element, so you can't use `action.click` with it (this will cause E2E failures)

Reply:
reverted